src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,100 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private int treshold;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        treshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+        size = 0;
```

Comment:
0 is the default value for int type, so if you want you can remove size initialization from constructor

Reply:
done